coderd/httpapi/httpapi.go

```diff
@@ -174,8 +173,7 @@ func WebsocketCloseSprintf(format string, vars ...any) string {
 	return msg
 }
 
-func ServerSentEventSender(rw http.ResponseWriter, r *http.Request) (func(ctx context.Context, sse codersdk.ServerSentEvent) error, error) {
-	var mu sync.Mutex
+func ServerSentEventSender(rw http.ResponseWriter, r *http.Request) (sendEvent func(ctx context.Context, sse codersdk.ServerSentEvent) error, closed chan struct{}, err error) {
```

Comment:
It seems like we can eliminate the error return from this as it's never set to a non-nil value

Reply:
Not really relevant for this PR, but that's a good point. I'd like to see it kept and remove the panic in favor of an error, though.